WeakAuras/Transmission.lua

```diff
@@ -1540,6 +1543,14 @@ function WeakAuras.ShowDisplayTooltip(data, children, matchInfo, icon, icons, im
       tinsert(tooltip, {1, L["It might not work correctly with your version!"], 1, 0, 0})
     end
 
+    if tocbuild and WeakAuras.IsClassic and tocbuild > 20000 then
```

Comment:
if this is classic WeakAuras, and the data does not have a tocbuild, then it must have been imported from an old import string. So:
```suggestion
    if (WeakAuras.IsClassic and (not tocbuild or tocbuild) > 20000 then
```